components/tabs/tab.js

```diff
@@ -21,18 +23,18 @@ class Tab extends TabMixin(LitElement) {
 
 	static get styles() {
 		const styles = [ css`
-			.d2l-tab-text {
+			.d2l-tab-inner-content {
```

Comment:
Should we be applying any whitespace between the slots and the text content, or is that up to the consumer? If so, hopefully we can find a nice way to do it conditionally without inspecting the contents of the slot. Perhaps a `flex` layout using `gap`, or maybe applying `margin` in a `::slotted` selector.

Also, if `d2l-icon` is used in the slot, should we be applying any style to it when the tab handle is selected, hovered, or focused? Starts to get a little opinionated, but I'm not sure - would the consumer have the ability to apply style in these states?

I guess some of this comes down to - what do we want `d2l-tab` to do out-of-the-box vs what should be as a custom tab impl?

![image](https://github.com/user-attachments/assets/98875110-1150-4b01-8590-be637f4d4a27)

Reply:
I was wondering about that. I left it with no styles so that the consumer could entirely handle that. That way they wouldn't have to be overriding anything if they didn't like what we had done. However, I'm open to adding some more out-of-the-box options. I'll see if Jeff has thoughts. If there are only a couple of ways this might be used (e.g., with d2l-icon and d2l-count-badge) then it likely makes sense for us to handle things.

Reply:
Sounds good. Yeah conv with Jeff sounds great.

Reply:
Strangely, the outline size in my screenshot above looks weird - maybe the result of the way I'd mocked it up. It looks ok in the latest demo.

It looks a little cramped when focused though.
![image](https://github.com/user-attachments/assets/95cb2677-884c-44d2-bf49-ae53987e2dfe)

Reply:
Yeah I was assuming the 2px padding we impose around the whole tab handle would include the new slot, rather than haning it outside of that padding. We can either refactor that so the padding is applied at a higher level that impacts both the text and the slots, or we can have the new slots impose their own 2px padding on the L/R side (depending which slot).